### PR TITLE
Hotfix 4.0.2

### PR DESCRIPTION
**Pull request by mikeminutillo**

### CODE REVIEW

File: src/ServiceControl.sln [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio Version 16
-VisualStudioVersion = 16.0.28803.452
+# Visual Studio 15
```
Comment (danielmarbach) [inline src/ServiceControl.sln]: Should we revert this?

File: src/ServiceControlInstaller.Packaging.UnitTests/DeploymentPackageTests.cs [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+                var mainEntries = zip.Entries.Where(x => x.FullName.StartsWith(deploymentPackage.ServiceName)).ToList();
+
+                CollectionAssert.IsNotEmpty(mainEntries, $"Expected a {deploymentPackage.ServiceName} folder in {deploymentPackage.FullName}");
+
+                CollectionAssert.IsEmpty(
+                    from mainEntry in mainEntries
+                    join entry in zip.Entries.Except(mainEntries) on mainEntry.Name equals entry.Name
+                    where entry.Length != mainEntry.Length
+                    select entry.FullName, 
+                    $"File sizes should match the ones in the {deploymentPackage.ServiceName} folder. Check versions of dependencies.");
```
Comment (danielmarbach) [inline src/ServiceControlInstaller.Packaging.UnitTests/DeploymentPackageTests.cs]: Is it sizes or file names?

Comment (mikeminutillo) [reply src/ServiceControlInstaller.Packaging.UnitTests/DeploymentPackageTests.cs]: The file names match. We're doing the join on file name. The sizes do not match.

So if you have `Newtonsoft.Json.dll` in the `ServiceControl` folder and in the `Transports/SqlServer` folder but the file sizes don't match, then we have a problem the files are obviously different. One of them will overwrite the other at install time.

File: src/ServiceControl.sln (hunk 1 above)
Comment (mikeminutillo) [reply src/ServiceControl.sln]: Done

**Pull request closed without merging**